dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -132,6 +132,11 @@
 			clock-frequency = <I2C_BITRATE_STANDARD>;
```

Comment:
rewrite a bit your commit title: "dts: espi: Add bindings for microchip XEC driver"